src/engine/debug/debug.h

```diff
@@ -3,7 +3,7 @@
 
 #include "lumix.h"
 #include "core/iallocator.h"
-#include "core/mt/sync.h"
+#include "core/MT/sync.h"
```

Comment:
I was thinking about changing all paths to lowercase. New code is already done that way, but it's ok to keep it upper case if it is more common on Linux.